Updated .gitignore, Checked if in CoLab, Checked if new Dataset and c…

MrSteve2 opened the pull request:
Detected if running in CoLab and if there is a New dataset

MrSteve2 committed:
Updated .gitignore, Checked if in CoLab, Checked if new Dataset and conditionally unzipped

MrSteve2 committed:
fix: modified to detect if in CoLab and if there is a new dataset

Modified notebook.ipynbi(cells 2-4) to
 - detect if we are running in CoLab
 - detect if there is a new dataset to be downloaded from Kaggle

Closes: Issue-13  https://github.com/VissuP/COVID-19-challenge/issues/11

Fireforge commented on notebook.ipynb:
```diff
@@ -1,283 +1,278 @@
+    "if (len(result) == 1) :\n",
+    "    NewDataset = False\n",
+    "else :\n",
+    "    NewDataset = True\n",
```
Fix this to be in a single line like: `new_dataset = len(result) != 1`

Fireforge commented on notebook.ipynb:
```diff
@@ -1,283 +1,278 @@
+   "source": [
+    "import zipfile\n",
+    "\n",
+    "if NewDataset :\n",
```
No space before colon

MrSteve2 committed:
fix(notebook.ipynb, .gitignore): Modified to detect CoLab and if there is a new Kaggle dataset

Modified notebook.ipynb to:
 - check if running in CoLab and safely handle if not
 - check if there is a new Kaggle dataset and update files in /data if necessary

CLOSES: Issue-11  https://github.com/VissuP/COVID-19-challenge/issues/11

MrSteve2 committed:
fix(notebook.ipynb): Added checks for CoLab and detecting new datasets

Modified to run in CoLab and locally.
Also check if there is a new kaggle dataset and if so to replace the current one.

Closes: Issue 11 https://github.com/VissuP/COVID-19-challenge/issues/11

Fireforge reviewed (approved):
👍
